### ISSUE TITLE
convert_tf_checkpoint_to_pytorch get different result?

### ISSUE THREAD

**Issue by light8lee** [huggingface/pytorch-transformers#634]
I'm using this repo to do Chinese NER on MSRA dataset, when I use the pretrained model `bert-base-chinese ` , the result is very good, it can reach 0.93+ f1 on test set in first epoch. But when I used `convert_tf_checkpoint_to_pytorch` to convert the original bert released checkpoint `chinese_L-12_H-768_A-12` to pytorch_model.bin, the result is very bad, it can only reach 0.5 f1 score on test set. Is there anything I do wrong?
And when I used `BertForPreTraining.from_pretrained`, the result is alright.
